[2024-05-06T04:54:10.118+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-06T04:54:10.327+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.update_entry manual__2024-05-06T04:53:55.730459+00:00 [queued]>
[2024-05-06T04:54:10.408+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.update_entry manual__2024-05-06T04:53:55.730459+00:00 [queued]>
[2024-05-06T04:54:10.412+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-06T04:54:10.508+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): update_entry> on 2024-05-06 04:53:55.730459+00:00
[2024-05-06T04:54:10.592+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'update_entry', 'manual__2024-05-06T04:53:55.730459+00:00', '--job-id', '10', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmp9pjn819g']
[2024-05-06T04:54:10.633+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask update_entry
[2024-05-06T04:54:10.634+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2280) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-06T04:54:10.639+0000] {standard_task_runner.py:63} INFO - Started process 2293 to run task
[2024-05-06T04:54:10.907+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.update_entry manual__2024-05-06T04:53:55.730459+00:00 [running]> on host 0cea9ba68f60
[2024-05-06T04:54:11.308+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='update_entry' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T04:53:55.730459+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T04:53:55.730459+00:00'
[2024-05-06T04:54:11.331+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-06T04:54:11.335+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-06T04:54:11.336+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo update_entry']
[2024-05-06T04:54:11.458+0000] {subprocess.py:86} INFO - Output:
[2024-05-06T04:54:11.479+0000] {subprocess.py:93} INFO - update_entry
[2024-05-06T04:54:11.480+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-06T04:54:11.481+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-06T04:54:11.563+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=update_entry, execution_date=20240506T045355, start_date=20240506T045410, end_date=20240506T045411
[2024-05-06T04:54:11.682+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-06T04:54:11.924+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-06T04:54:11.963+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
